### ISSUE TITLE
is Peatio a proper solution for DPOS / BTC pair trading?

### ISSUE THREAD

**Issue by kankando** [InfraexDev/peatio#48]
We are looking for a solution to integrate DPOS / BTC coin pair to be traded with Peatio, see if anyone have any experience on this. Thanks

**Comment by amarian12** [comment]
yes peatio can do it, here is custom work:  https://www.nlexch.com

**Comment by rowdyruckus** [comment]
1,000,000 ESB bounty for connecting with EOS native coins!
